src/dll/edbgEcmdDllInfo.C

```diff
@@ -41,10 +41,10 @@ uint32_t dllQueryDllInfo(ecmdDllInfo & o_dllInfo) {
   o_dllInfo.dllProduct = "POWER9";
   //o_dllInfo.dllType = "edbg";
   //o_dllInfo.dllProduct = "OpenPOWER";
-  o_dllInfo.dllProductType = GIT_COMMIT_REV;
```

Comment:
This change cannot get checked in.  While meson build may not have a way to add these variables yet, the default build does and would break functionality if removed.

Reply:
Yes will do

Reply:
done. found a way meson can do it